media/js/newsletter/newsletter-init.es6.js

```diff
@@ -3,9 +3,43 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
+/* eslint-disable no-console */
 
 import MzpNewsletter from '@mozilla-protocol/core/protocol/js/newsletter';
-import NewsletterForm from './newsletter.es6';
 
-MzpNewsletter.init();
-NewsletterForm.init();
+const successCustomCallback = () => {
+    const newsletters = Array.from(
+        document.querySelectorAll("input[name='newsletters']:checked")
```

Comment:
Nit: suggest making this selector a little less broad?
```suggestion
        document.querySelectorAll(".mzp-c-newsletter-form input[name='newsletters']:checked")
```